### PR TITLE
Add notification for modification

### PR DESCRIPTION
**Pull request by bendaamerahm**
add notification for modification (hypothesis)

### CODE REVIEW

**Review by JacquesBorsenberger — APPROVED** [review]

File: src/main/java/org/gridsuite/study/server/StudyService.java [hunk 1]
```diff
@@ -86,6 +86,7 @@ public class StudyService {
 
     static final String HEADER_USER_ID = "userId";
     static final String HEADER_STUDY_UUID = "studyUuid";
+    public static final String HEADER_PARENT_NODE = "parentNode";
```
Comment (sBouzols) [inline src/main/java/org/gridsuite/study/server/StudyService.java:89]: why `public` in comparison of others strings here ?

File: src/main/java/org/gridsuite/study/server/StudyService.java [hunk 2]
```diff
@@ -120,6 +121,10 @@ public class StudyService {
 
     static final String FIRST_VARIANT_ID = "first_variant_id";
 
+    public static final String MODIFICATIONS_CREATING_IN_PROGRESS = "creatingInProgress";
```
Comment (sBouzols) [inline src/main/java/org/gridsuite/study/server/StudyService.java:124]: why `public` access modifier here

File: src/test/java/org/gridsuite/study/server/StudyTest.java [hunk 3]
```diff
@@ -2747,6 +2753,33 @@ private void checkEquipmentMessagesReceived(UUID studyNameUserIdUuid, UUID nodeU
...
+        assertEquals(studyNameUserIdUuid, headersStudyUpdate.get(StudyService.HEADER_STUDY_UUID));
+        assertEquals(MODIFICATIONS_UPDATING_IN_PROGRESS, headersStudyUpdate.get(StudyService.HEADER_UPDATE_TYPE));
+    }
+
+    private void checkEquipmentDeletingMessagesReceived(UUID studyNameUserIdUuid, UUID nodeUuid) {
+        // assert that the broker message has been sent for updating study type
+        Message<byte[]> messageStudyUpdate = output.receive(TIMEOUT);
+        assertEquals("", new String(messageStudyUpdate.getPayload()));
+        MessageHeaders headersStudyUpdate = messageStudyUpdate.getHeaders();
+        assertEquals(studyNameUserIdUuid, headersStudyUpdate.get(StudyService.HEADER_STUDY_UUID));
```
Comment (sBouzols) [inline src/test/java/org/gridsuite/study/server/StudyTest.java:2779]: You should add an assert over HEADER_NODE_UUID here no ?

**Review by sBouzols — CHANGES_REQUESTED** [review]
almost ok

File: src/test/java/org/gridsuite/study/server/StudyTest.java (hunk 3 above)
Comment (bendaamerahm) [reply src/test/java/org/gridsuite/study/server/StudyTest.java:2779]: yes , for my case i need to do assert on HEADER_PARENT_NODE

File: src/main/java/org/gridsuite/study/server/StudyService.java (hunk 2 above)
Comment (bendaamerahm) [reply src/main/java/org/gridsuite/study/server/StudyService.java:124]: my bad , ill correct that

**Review by sBouzols — APPROVED** [review]
Code Review OK
Will test with front

**Review by sBouzols — APPROVED** [review]
Code Review OK
Tests OK

**Review by JacquesBorsenberger — APPROVED** [review]

**Pull request merged**